Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.16-buster
+FROM node:14.17-buster
```

Comment:
Is it worth matching this up with the circle config one too?

Reply:
Well I suppose so, but it depends on whether the hmpps executor supports that version. I'll try this first